test/helper.ts

```diff
@@ -33,3 +35,17 @@ export function parseAndProcess(input: string, fn: (ast: ESTree.Program) => void
   fn(ast);
   return astring(ast).trim();
 }
+
+const defaultSandbox = {
+  console,
+  module: {
+    exports: {}
+  },
+  require
+};
+
+export function virtualModule(code: string, optionsSandbox = {}) {
```

Comment:
tsc is fine with, so I tend to be, too. Typings are for the machines, right? I don't see too much value in typing a deep merge objects – especially as this already has a broad structural definition in `defaultSandbox`. Writing down the indexable ts manierism here would hold little to no additonal information for humans.